scripts/ci-wrapper

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+sudo ./scripts/bootstrap
+git fetch origin ${1}
+git checkout origin/${1} ./assets/index.yaml || true
```

Comment:
fetch `./assets/index.yaml` and `./sha256sum` from asset branch. This is because dev-v2.5 branch will have no knowledge against what previous index.yaml we have and we will lose all the info on old helm chart. `./sha256sum` on asset branch is the last-run sha256sum of each folder in order to compare what's change in this new commit.

Reply:
@aiyengar2  It can't be clean up as `assets/` and `sha256sum/` has to be copied over to asset branch. Also it doesn't really need to be cleaned up since this script should never be run through a developer.